src/utilities/configuration.ts

```diff
@@ -23,7 +23,18 @@ if (!subscan.network) {
 if (!subscan.secret) {
   throw new ConfigurationError('No subscan secret provided');
 }
-
+const indexer = {
+  graphqlEndpoint: import.meta.env.GRAPHQL_ENDPOINT as string,
+  polkadotRPCEndpoint: import.meta.env.POLKADOT_RPC_ENDPOINT as string,
+};
+if (!indexer.graphqlEndpoint) {
+  throw new ConfigurationError('No endpoint for the GraphQL server provided');
+}
+if (!indexer.polkadotRPCEndpoint) {
+  throw new ConfigurationError(
+    'No R.P.C. endpoint for the polkadot.js explorer provided',
```

Comment:
I just now realized that while this is an American style of abbreviations, I’ve never seen American developers use it, and they are very populous. Curious!

Reply:
This is only "correct" in US, this is why I call it "American"

Reply:
It's getting obsolete. I just read about it. ;( I learned that this was the "correct" way in primary school.